### PR TITLE
WRN-12676: Cross-Platform Enact Android: Picker - tall string is truncated

### PR DESCRIPTION
**Pull request by alexandrumorariu**
### Checklist

* [X] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [X] At least one test case is included for this feature or bug fix
* [X] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Changed the font for picker Item and Button to Noto Sans to display the tall glyphs correctly.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)
The font was applied only to pickerItem and Button because it had no impact on the rest of the components. The change of fonts does not fix the problem for joined picker.

### Links
[//]: # (Related issues, references)
WRN-12676

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ The following is a curated list of changes in the Enact sandstone module, newest
 
 ### Fixed
 
+- `sandstone/Picker` changed the size of @sand-picker-line-height
```
Comment (daniel-stoian-lgp) [inline CHANGELOG.md:12]:
```suggestion
- `sandstone/Picker` line-height so tall characters are not truncated on mobile devices
```

**Review by daniel-stoian-lgp — APPROVED** [review]
LGTM, but someone else must approve

**Comment by daniel-stoian-lgp** [comment]
Seems that the difference between PC and Mobile device are the different font families available on the device

Mobile vs PC:

image

**Comment by adrian-cocoara-lgp** [comment]
I think the changelog should be moved to 'Changed' category

**Comment by alexandrumorariu** [comment]
> I think the changelog should be moved to 'Changed' category



> 

The problem is treated as a bug that's why it is marked as a fix not a feature change.

**Comment by jeonghee27** [comment]
This issue will be solved on https://github.com/enactjs/enact/pull/2990

**Pull request closed without merging**